addons/nametags/CfgFactionClasses.hpp

```diff
@@ -0,0 +1,23 @@
+class CfgFactionClasses {
+    class OPF_F {
+        GVAR(rankIcons)[] = {QPATHTOF(UI\icons_russia\private_gs.paa), QPATHTOF(UI\icons_russia\corporal_gs.paa), QPATHTOF(UI\icons_russia\sergeant_gs.paa), QPATHTOF(UI\icons_russia\lieutenant_gs.paa), QPATHTOF(UI\icons_russia\captain_gs.paa), QPATHTOF(UI\icons_russia\major_gs.paa), QPATHTOF(UI\icons_russia\colonel_gs.paa)};
```

Comment:
```suggestion
        GVAR(rankIcons)[] = {
            QPATHTOF(UI\icons_russia\private_gs.paa),
            QPATHTOF(UI\icons_russia\corporal_gs.paa),
            QPATHTOF(UI\icons_russia\sergeant_gs.paa),
            QPATHTOF(UI\icons_russia\lieutenant_gs.paa),
            QPATHTOF(UI\icons_russia\captain_gs.paa),
            QPATHTOF(UI\icons_russia\major_gs.paa),
            QPATHTOF(UI\icons_russia\colonel_gs.paa)
        };
```
Just an idea, we don't need to conserve vertical space here I'd say?

Reply:
I suggest macro like RANK_ICONS(russia) which would contain that rank array

Reply:
Yeah if the filename scheme is the same for everything.

Reply:
It's fine after dedmen's edit.